wagtail_shortcode/handlers.py

```diff
@@ -0,0 +1,38 @@
+from draftjs_exporter.dom import DOM
+from wagtail.admin.rich_text.converters.html_to_contentstate import (
+    InlineEntityElementHandler,
+)
+
+
+SHORTCODE_ANCHOR_TARGET_IDENTIFIER = "shortcode"  # Used for linktype attribute
+
+
+def shortcode_entity_decorator(props):
+    """
+    Draft.js ContentState to database HTML.
+    Converts the SHORTCODE entities into a anchor tag, which will then
+    be further altered by a custom LinkHandler.
+    """
+    return DOM.create_element(
+        "a",
+        {
+            "shortcode": props.get("shortcode"),
```

Comment:
Looks like you're storing it in the `shortcode` attribute rather than the `href`, which they originally suggested on the call. IMO this is fine, but worth double checking this afternoon with them that the db format for a shortcode will be x, any conversion to FE will be done by the `LinkHandler` + additional python code they write.

Reply:
Thats a good point. 

I figured it could cause less confusion between creating the entity on the front-end (`createEntity()` in main.tsx), then passing that value as a prop here (`shortcode_entity_decorator`) and then getting the value from ShortcodeEntityElementHandler again to the content state.

However I've gone with href now as it's probably better to maintain a database state closer to what they're expecting. To try and tackle any potential confusion I've added comments and labeled variables like so:
```
"href": props.get("href"),  # the shortcode value
...
href: shortcodeValue,
...
const shortcodeValue = data.href || null;
```

Reply:
Scratch the above, using href conflicts with wagtail's default external link handler/entity, use of the `shortcode` attribute seems like the safer option.